packages/ui/src/common/utils.ts

```diff
@@ -3,3 +3,12 @@ export const isFunction = (something: unknown): something is CallableFunction =>
 export const isDefined = <T extends any>(something: T | undefined): something is T => typeof something !== 'undefined'
 
 export const isNumber = (something: unknown): something is number => typeof something === 'number'
+
+export const objectShallowContains = <T extends { [k: string]: any }>(contained: T): ((container: T) => boolean) => {
```

Comment:
I think that the name might be misleading. AFAICS it compares an object to be shallowly equal to others, so maybe just `objectEquals()`? 

Below some variables names hint, but you can use any other you see fit.

```suggestion
export const objectEquals = <T extends Record<string, any>}>(reference: T): ((compared: T) => boolean) => {
```

Reply:
Hi @jodator, thanks for the review.

Actually it was only doing something similar to Jest  [`expect.objectContaining`](https://jestjs.io/docs/expect#expectobjectcontainingobject). But I guess it makes more sense to check the keys length of the `compared` object to have something easier and safer to reuse somewhere else. So I changed the logic a bit.